### PR TITLE
Botgarden propagation KS changes

### PR DESCRIPTION
**Pull request by kspurgin**

### CODE REVIEW

File: spec/fixtures/files/datahashes/botgarden/propagation1.json [hunk 1]
```diff
@@ -5,14 +5,14 @@
     "propDate": "2020-11-09",
     "propReason": "conservation",
     "propComments": "Be happy",
-    "scarStrat": "cold strat^^boiling water^^",
-    "duration": "25^^10",
-    "durationUnit": "Hours^^Days^^",
```
Comment (kspurgin) [inline spec/fixtures/files/datahashes/botgarden/propagation1.json:10]: This is a repeating field GROUP (without repeating field SUBGROUP), so we use `;` delimiter

File: spec/fixtures/files/datahashes/botgarden/propagation1.json [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
+    "scarStrat": "cold strat;boiling water;",
+    "duration": "25;;10",
+    "durationUnit": "Hours;Days;",
     "extraSeeds": "true",
     "spores": "true",
-    "cuttingType": "hardwood^^",
-    "hormone": "hormone^^",
-    "concentration": "Worry^^",
```
Comment (kspurgin) [inline spec/fixtures/files/datahashes/botgarden/propagation1.json:15]: These fields are not repeatable, so use no delimiter

**Review by kspurgin — COMMENTED** [review]
@kevin-kariuki - please review changes and merge into #48

File: spec/fixtures/files/datahashes/botgarden/propagation1.json (hunk 2 above)
Comment (kevin-kariuki) [reply spec/fixtures/files/datahashes/botgarden/propagation1.json:15]: Noted.

File: spec/fixtures/files/datahashes/botgarden/propagation1.json (hunk 1 above)
Comment (kevin-kariuki) [reply spec/fixtures/files/datahashes/botgarden/propagation1.json:10]: Noted.

**Pull request merged**